CONTRIBUTING.rst

```diff
@@ -561,6 +587,8 @@ Build the documentation on Linux or Mac OS with:
 
 Build the documentation on Windows with:
 
+.. code::
```

Comment:
Explicit is better than implicit.
```suggestion
.. code::bash
```

Reply:
Bash is not very windowsy though 😛 Is there syntax highlighting for batch?

Reply:
Opps. I didn't notice that it is window.
```suggestion
.. code::winbatch
```
ref https://pygments.org/languages/

Reply:
Modified your suggestion to include a space to be aligned with the sphinx documentation:
https://www.sphinx-doc.org/en/master/usage/restructuredtext/directives.html#directive-code-block